src/api/burn/balutil/inc/IBootstrapperApplication.h

```diff
@@ -488,27 +488,19 @@ DECLARE_INTERFACE_IID_(IBootstrapperApplication, IUnknown, "53C31D56-49C0-426B-A
         __inout int* pResult
         ) = 0;
 
-    // OnExecuteFilesInUse - called when the engine encounters files in use while
-    //                       executing a package.
+    // OnExecuteFilesInUse - called when the engine receives a files in use message
+    //                       while executing a package.
     //
-    // Return:
```

Comment:
Why remove these? This is the only "doc" without spelunking source.

Reply:
You mean source code? Burn doesn't own how the return codes are processed. The BA needs to look at the source of the FilesInUse message and return a value based on that source's documentation which is provided as a link here.

Reply:
I didn't realize it was just pass-through. Hopefully we'll never need more than an int...

Reply:
I guess it's not exactly pass-through, there's still some coercion by calling `UserExperienceCheckExecuteResult`. I wasn't sure whether to remove that, too.

Reply:
Seems that it should be completely pass-through (though we still want to be able to talk to Burn) or translate Burn return values to the underlying engine (as Burn is hard-coded for the ones it supports).